test/e2e_node/jenkins/benchmark/benchmark-config.yaml

```diff
@@ -0,0 +1,50 @@
+---
+images:
+  containervm-density1:
+    image: e2e-node-containervm-v20160321-image
```

Comment:
Shouldn't image here be "containervm-v20160321-image"? Isn't "e2e-node-containervm-v20160321-image" is the node name created by node-e2e for image of "containervm-v20160321-image"?